src/gui/Workbench/ScreenArea/FormScreen.module.css

```
@@ -37,12 +37,12 @@ along with ORIGAM. If not, see <http://www.gnu.org/licenses/>.
   padding: 10px;
   flex-shrink: 0;
   flex-grow: 0;
-  /* border: 1px solid black;*/
+  /* border: 1px solidvar(--black);*/
```

Yes, it is commented out, but....

```suggestion
  /* border: 1px solid var(--black);*/
```